### PR TITLE
Rotate test etcd CA and certs

### LINKED ISSUE #2749: etcd tests fail due to expired certificate
### Description
As seen at https://drone.platform.teleport.sh/gravitational/gravity/734/1/8

```
FAIL: etcd_test.go:114: ESuite.TearDownTest

etcd_test.go:120:
    c.Assert(err, IsNil)
... value *trace.TraceErr = 
ERROR REPORT:
Original Error: x509.CertificateInvalidError x509: certificate has expired or is not yet valid: current time 2022-09-20T21:16:32Z is after 2022-07-08T20:17:00Z
Stack Trace:
	/gopath/src/github.com/gravitational/gravity/lib/storage/keyval/etcd_test.go:64 github.com/gravitational/gravity/lib/storage/keyval.(*tempBackend).Delete
	/gopath/src/github.com/gravitational/gravity/lib/storage/keyval/etcd_test.go:116 github.com/gravitational/gravity/lib/storage/keyval.(*ESuite).TearDownTest
	/go/src/reflect/value.go:543 reflect.Value.call
	/go/src/reflect/value.go:339 reflect.Value.Call
	/gopath/src/github.com/gravitational/gravity/vendor/gopkg.in/check.v1/check.go:730 gopkg.in/check%2ev1.(*suiteRunner).runFixture.func1
	/gopath/src/github.com/gravitational/gravity/vendor/gopkg.in/check.v1/check.go:675 gopkg.in/check%2ev1.(*suiteRunner).forkCall.func1
	/go/src/runtime/asm_amd64.s:1581 runtime.goexit
User Message: x509: certificate has expired or is not yet valid: current time 2022-09-20T21:16:32Z is after 2022-07-08T20:17:00Z ("x509: certificate has expired or is not yet valid: current time 2022-09-20T21:16:32Z is after 2022-07-08T20:17:00Z")

```

### PR DESCRIPTION
**Pull request by wadells**
## Description
These expired a month ago or so and all test are failing as described in https://github.com/gravitational/gravity/issues/2749

I also improved the Makefile to make rotating these certificates easier in the future.  Not that this is likely to come up again -- but it helped me test run the changes.

## Type of change
* Regression fix (non-breaking change which fixes a regression)
* Internal change (not necessarily a bug fix or a new feature)

## Linked tickets and other PRs
* Fixes https://github.com/gravitational/gravity/issues/2749
* Needs backports to any branch that will be accepting PRs

## TODOs
- [x] Self-review the change
- [x] Make sure the drone unit tests pass
- [x] Address review feedback

### CODE REVIEW

File: assets/certs/Makefile [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+ca: ca.crt
+
+%.key:
+	openssl genrsa -out $@ 2048
+
+ca.csr: ca.key ca.config
+	openssl req -new -key $< -out $@ -config ca.config
+
+ca.crt: ca.csr ca.key
+	cat ca.csr | openssl x509 -req -days 1095 -sha256 -keyform pem -signkey ca.key -set_serial 0x2EC9165BAC1CFD93B558790F8EB291D99E359072 -out ca.crt
```
Comment (wadells) [inline assets/certs/Makefile:16]: `0x2EC9165BAC1CFD93B558790F8EB291D99E359072` was pulled off the existing CA certificate.

File: assets/certs/ca.config [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[req]
+distinguished_name = req_distinguished_name
+prompt = no
+[req_distinguished_name]
+C = US
+ST = California
+L = San Francisco
+O = "Honest Achmed's Used Certificates"
+OU = Hastily-Generated Values Divison
```
Comment (reedloden) [inline assets/certs/ca.config:9]:
```suggestion
OU = Hastily-Generated Values Division
```

Comment (wadells) [reply assets/certs/ca.config:9]: This was pulled off the existing certificate.  I can update it but that'd also churn the CSR, which I left untouched.

Comment (reedloden) [reply assets/certs/ca.config:9]: No worries. I just notice these things easily. :-)

**Review by bernardjkim — APPROVED** [review]

**Comment by wadells** [comment]
Unit tests came back clear: 

https://drone.platform.teleport.sh/gravitational/gravity/737/1/8

:crossed_fingers: for robotest

**Pull request merged**